ansible/configs/ocp4-workshop/pre_software.yml

```diff
@@ -45,6 +45,10 @@
   become: true
   roles:
   - { role: "bastion",              when: 'install_bastion | bool' }
+  - role: "status-report-install"
```

Comment:
This must belong in the status-report PR.

Reply:
Do we make the status report API dependent on this change or the other way around?

Reply:
We can do the status-report api in first.

Reply:
To clarify (because the above comments might be confusing), we can leave this here but should get the status report api merged first.

Reply:
I suggest going with the reporting API first since other roles implement it as well, not the other way around.